app/api-v1/services/identityService.js

```diff
@@ -24,9 +24,29 @@ const getMemberByAlias = async (req, alias) => {
   throw new InternalError({ message: 'Internal server error' })
 }
 
+const getMemberBySelf = async (req, alias) => {
+  const response = await fetch(`${URL_PREFIX}/self/${encodeURIComponent(alias)}`, {
+    headers: {
+      Authorization: `Bearer ${req.token}`,
+    },
+  })
+
+  if (response.status === 404) {
+    throw new BadRequestError({ message: `Self does not exist` })
+  }
+
+  if (response.ok) {
```

Comment:
I think this check is not needed and we can return a promise `response.json()` and if it throws than an error middleware will handle it with 500 as well